src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -295,6 +305,19 @@ public function configureOptions(OptionsResolver $resolver)
             return $options['expanded'];
         };
 
+        // for BC with the "translation_domain" option
```

Comment:
I would remove that comment. we add "BC" usually for deprecated BC. but in this case it's part of how it is supposed to be.